src/routes/news/[slug].svelte

```diff
@@ -67,8 +101,8 @@
 
         <img
                 class="rounded-xl mb-6 w-full"
-                alt={article.articleHeroImage.title}
-                src={article.articleHeroImage.url}
+                alt={article.articleHeroImage?.title}
+                src={article.articleHeroImage?.url}
         />
```

Comment:
It is best not to render the image element altogether if the `url` property is `undefined`.